packages/docs-reanimated/docs/debugging/logger-configuration.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: logger-configuration
+sidebar_label: Logger configuration
+sidebar_position: 2
+---
+
+# Logger configuration
+
+Reanimated shows warnings that indicate misuses of the library API, such as modifying the [shared value](/docs/fundamentals/glossary#shared-value) during component re-render. These logs can be configured to be more or less verbose. The **default** configuration shows **all warnings and errors**. You can change this behavior by using the `configureReanimatedLogger` function.
+
+## Reference
+
+To configure the Reanimated logger, import `configureReanimatedLogger` from `react-native-reanimated` and call it with the desired configuration.
+
+```js
+import {
+  configureReanimatedLogger,
+  ReanimatedLogLevel,
+} from 'react-native-reanimated';
+
+// This is a default configuration
+configureReanimatedLogger({
+  level: ReanimatedLogLevel.warn,
+  strict: true, // Reanimated runs in strict mode by default
+});
+```
+
+<details>
+<summary>Type definitions</summary>
+
+```typescript
+function configureReanimatedLogger(config: LoggerConfig): void;
+
+type LoggerConfig = {
+  level?: ReanimatedLogLevel;
+  strict?: boolean;
+};
+
+enum ReanimatedLogLevel {
+  warn = 1,
+  error = 2,
+}
+```
+
+</details>
+
+### Configuration options
+
+#### `level`
+
+A value od the `ReanimatedLogLevel` enum that defines the minimum level of the logs that will be shown.
+
+#### `strict`
+
+A boolean value that enables or disables strict mode. When strict mode is enabled, Reanimated will show more warnings that can help you to catch potential issues in your code.
+
+## Remarks
+
+- The logger configuration is global and affects all Reanimated animations in your app.
```

Comment:
> of your code indiscriminately

Doesn't it suggest that this may affect user's code that is not related to Reanimated?
You suggested me before to be more explicit that changes affect how Reanimated work. not how the app works in general.

I can agree that `animations` word might be confusing and should be changed to something else.

Reply:
> Doesn't it suggest that this may affect user's code that is not related to Reanimated?
Your right, it might suggest that.

> The logger configuration is global and affects all warnings and errors shown by Reanimated. There's no option to configure the logger per file/component.
Me likes 👍